ssh-krb-node-executor-plugin/contents/scp-krb-executor.sh

```diff
@@ -13,34 +13,41 @@ shift
 SRC_FILE=$1
 shift
 DEST_FILE=$1
-KERB_KEYTAB=$RD_CONFIG_KERBEROS_KEYTAB
 
-if [ ! -f "$KERB_KEYTAB" ]; then
-  echo "Keytab $KERB_KEYTAB not found" >&2
+
+if [ ! -f "$RD_CONFIG_KERBEROS_KEYTAB" ]; then
+  echo "Keytab $RD_CONFIG_KERBEROS_KEYTAB not found" >&2
   exit 2
 fi
 
 # random delay (0..0.5s) to make it work with parallel exec
 sleep "$(bc <<< "scale=2; $(printf '0.%02d' $(( RANDOM % 100))) / 2")"
 
-# status 1 if the credentials cache cannot be read or is expired, and with status 0 otherwise
-klist -s
-if [ $? -ne 0 ]; then
-  kinit -kt "$KERB_KEYTAB" "$RD_CONFIG_KERBEROS_USER"
 
-  # verify ticket has been successfuly created
-  klist -s
-  if [ $? -ne 0 ]; then
-    echo "Kinit failure when calling kinit -kt $KERB_KEYTAB $RD_CONFIG_KERBEROS_USER" >&2
+# override the default kerberos cache file if required
+if [[ "$RD_CONFIG_USE_KERBEROS_CUSTOM_CACHE_FILE" == 'true' ]]; then
```

Comment:
This seems to be the construct to be used to compare, according to the [Bash programming howto](https://tldp.org/HOWTO/Bash-Prog-Intro-HOWTO-6.html).

```suggestion
if [ "$RD_CONFIG_USE_KERBEROS_CUSTOM_CACHE_FILE" = 'true' ]; then
```

Is there any particular reason to use the other form?

Reply:
No as the test is done against a literal string.
But it's using the same format as the last `if` statement :
```bash
if [[ ${RD_CONFIG_DO_KDESTROY} == 'true' ]]; then
  kdestroy
fi
```

Done (b531295).